docker/ubuntu/koandroid/Makefile

```diff
@@ -1,10 +1,22 @@
-VERSION=0.5.3
+VERSION=0.7.0
```

Comment:
Bumped to 0.7.0, using 20.04, but there's a 0.6.0 using 22.04… It's a little bit confusing, and hard to know what your getting when looking at the available images on docker hub. Maybe it would make more sense to add the base version to the image name? (like for kobase)

Reply:
> It's a little bit confusing, and hard to know what you're getting when looking at the available images on docker hub. Maybe it would make more sense to add the base version to the image name? (like for kobase)

Perhaps the naming scheme should be adjusted as well as more care taken with it (e.g., always use a `-dev` suffix for testing,[1] although that wouldn't have helped in this particular case). It's basically just been me messing around a bit in here. :-) All of which is a long way of saying that 0.6.0 is can be safely overwritten. In this case it stuck around due to #85. What happened is that I tested it locally, everything worked so I pushed it, and only then realized it didn't work in CircleCI and GitLab CI.

But all that aside really. Something like `20.04-1.0` perhaps?

[1] Though to be clear, testing is something to be done locally. The testing I'm referring to here has normally just been a formality.